src/styles.scss

```diff
@@ -3,3 +3,24 @@
 @tailwind utilities;
 
 @import "bootstrap-icons";
+
+@font-face {
+  font-family: 'FixelDisplay-Bold';
+  src: url('/assets/fonts/FixelDisplay-Bold.woff2') format('woff2');
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'FixelDisplay-Medium';
+  src: url('/assets/fonts/FixelDisplay-Medium.woff2') format('woff2');
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'FixelDisplay-Regular';
+  src: url('/assets/fonts/FixelDisplay-Regular.woff2') format('woff2');
```

Comment:
replace the line with:  `src: url('./assets/fonts/FixelDisplay-Regular.woff2') format('woff2');`

Reply:
@Viacheslav-Chopyk

Reply:
done